### PR TITLE
Create upload.html

### PR DESCRIPTION
**Pull request by mayank-ch**
Master server with retry feature.
Sending response protocol buffer file back to user.

### CODE REVIEW

File: master_server/Request.proto [hunk 1]
```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
```
Comment (kaladin-google) [inline master_server/Request.proto:1]: I have left comments on proto formatting in Neeraj's change. Please follow the changes he will make.

File: master_server/master_server.py [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+      response = requests.post(url=vm_n, files={'task_request':
+          task_request.SerializeToString()})
+      return 'file uploaded successfully,your request_id is '+ \
+          str(requ.request_id)+'  '+' address is '+str(requ.last_vm_assigned)+ \
+          ' number of retries allowed is '+str(requ.number_of_retries_allowed)
+    except:
+      return 'try again later'
+
+@app.route('/vm_status', methods=['GET', 'POST'])
+def upload_fi():
```
Comment (kaladin-google) [inline master_server/master_server.py:145]: upload_file*

Comment (kaladin-google) [reply master_server/master_server.py:145]: I see that is already used, please use more descriptive function names.

Comment (mayank-ch) [reply master_server/master_server.py]: Changes have been made as suggested.

File: master_server/Request.proto (hunk 1 above)
Comment (mayank-ch) [reply master_server/Request.proto:1]: Changes have been made as suggested.

**Pull request closed without merging**